bazel/repository_locations.bzl

```diff
@@ -955,11 +955,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Webassembly Micro Runtime",
         project_desc = "A standalone runtime with a small footprint for WebAssembly",
         project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
-        version = "WAMR-05-18-2022",
-        sha256 = "c71f571f99c7028fd7595126ec25d6df4bc9ba62702bb88cbd5ea8a29e477f12",
+        version = "c3d66f916ef8093e5c8cacf3329ed968f807cf58",
```

Comment:
Any reason why the latest release `WAMR-1.1.2` wasn't used? It contains this commit sha. /cc @wrowe

Reply:
No special reason. Only because there is no *WAMR-1.1.2*(Dec.16) tag when creating this PR (Nov.24). Another bumping version PR will be filed soon.

Reply:
BTW, do I have to keep the *same version* in both *envoy* and *proxy-wasm-cpp-host*?

Reply:
Unfortunately `proxy_wasm_cpp_host` doesn't ship versions. +1 on bumping to the release as a follow up.